cli/cmd/diagnose_util/crd_utils.go

```diff
@@ -0,0 +1,150 @@
+package diagnose_util
+
+import (
+	"compress/gzip"
+	"context"
+	"fmt"
+	"github.com/odigos-io/odigos/cli/pkg/kube"
+	"github.com/odigos-io/odigos/k8sutils/pkg/client"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"os"
+	"path/filepath"
+	"sigs.k8s.io/yaml"
+	"sync"
+)
+
+const (
+	CRDName         = "crdName"
+	CRDGroup        = "crdGroup"
+	actionGroupName = "actions.odigos.io"
+	odigosGroupName = "odigos.io"
+)
+
+var CRDsList = []map[string]string{
+	{
+		CRDName:  "addclusterinfos",
+		CRDGroup: actionGroupName,
+	},
+	{
+		CRDName:  "deleteattributes",
+		CRDGroup: actionGroupName,
+	},
+	{
+		CRDName:  "renameattributes",
+		CRDGroup: actionGroupName,
+	},
+	{
+		CRDName:  "probabilisticsamplers",
+		CRDGroup: actionGroupName,
+	},
+	{
+		CRDName:  "piimaskings",
+		CRDGroup: actionGroupName,
+	},
+	{
+		CRDName:  "latencysamplers",
+		CRDGroup: actionGroupName,
+	},
+	{
+		CRDName:  "errorsamplers",
+		CRDGroup: actionGroupName,
+	},
+	{
+		CRDName:  "instrumentedapplications",
+		CRDGroup: odigosGroupName,
+	},
+	{
+		CRDName:  "instrumentationconfigs",
+		CRDGroup: odigosGroupName,
+	},
+	{
+		CRDName:  "instrumentationrules",
+		CRDGroup: odigosGroupName,
+	},
+	{
+		CRDName:  "instrumentationinstances",
+		CRDGroup: odigosGroupName,
```

Comment:
Can you please add destinations as we discussed?

Reply:
sure sure, I separated this task for different PRs